CHANGELOG.md

```diff
@@ -11,6 +11,18 @@ for further information about branching and tagging conventions.
 ### Added
 -  Added is_coinpayments checkbox on asset creation page
 
+### Changed
+- Hide input placeholder only if label prop is empty
```

Comment:
Hide input placeholder only if label prop is _not_ empty

Reply:
Renamed